remove references to deprecated sequence keyed by group interface (#373)
python/importlib_metadata: docs/using.rst

    >>> version('wheel')
    '0.32.3'

> You can also get a collection of entry points selectable by group, such as

Author: it's actually selectable by any of the EntryPoint attributes

```suggestion
You can also get a collection of entry points selectable by name, group or value, such as
```

Reviewer: I'd probably not want to emphasize by value, as I don't expect legitimate use-cases to care about select by value. By name on the other hand seems a likely usage. Maybe.

```suggestion
You can also get a collection of entry points selectable by group or name (or other attribute), such as
```

Author: Or other attribute is a bit odd when there's only one other attribute

Reviewer: Other possible attributes include `dist`, `module`, `attr`, and `extras`. See dbe114cbdc49.
```suggestion
You can also get a collection of entry points selectable by properties of the EntryPoint (typically 'group' or 'name'), such as
```

    >>> eps = entry_points()

The ``entry_points()`` function returns an ``EntryPoints`` object,

> a collection of all ``EntryPoint`` objects with ``names`` and ``groups``

Author: the sequence interface is deprecated:

```
DeprecationWarning: Accessing entry points by index is deprecated. Cast to tuple if needed.
```
